### ISSUE TITLE
Odoo 9 currency duplicate erroe

### ISSUE THREAD

**Issue by umarif** [odoo/odoo#11036]
Impacted versions: Odoo 9 free ERP

Steps to reproduce: there's a bug in currency module, getting an error when duplicate my currency
```
Current behavior:Odoo
Odoo Server Error
Traceback (most recent call last):
  File "C:\Odoo\server\.\openerp\http.py", line 643, in _handle_exception
  File "C:\Odoo\server\.\openerp\http.py", line 680, in dispatch
  File "C:\Odoo\server\.\openerp\http.py", line 316, in _call_function
  File "C:\Odoo\server\.\openerp\service\model.py", line 118, in wrapper
  File "C:\Odoo\server\.\openerp\http.py", line 309, in checked_call
  File "C:\Odoo\server\.\openerp\http.py", line 959, in __call__
  File "C:\Odoo\server\.\openerp\http.py", line 509, in response_wrap
  File "C:\Odoo\server\openerp\addons\web\controllers\main.py", line 892, in call_kw
  File "C:\Odoo\server\openerp\addons\web\controllers\main.py", line 884, in _call_kw
  File "C:\Odoo\server\.\openerp\api.py", line 250, in wrapper
  File "C:\Odoo\server\openerp\addons\base\res\res_currency.py", line 112, in copy
NameError: global name '_' is not defined
```
Expected behavior: must be a duplicate currency

Video/Screenshot link (optional):

**Comment by Yenthe666** [comment]
Hi @umarif,

I've just tested this and can confirm that this is a bug.
@mart-e pinging you for this one.
![image](https://cloud.githubusercontent.com/assets/6352350/15435042/680d6c94-1eba-11e6-8b85-48354b2a9698.png)

**Comment by mart-e** [comment]
Indeed I confirm the issue, this was due to a bad forward port of d780f9476d43eb4fd25c852dd4e691002f4a6dcf in 9.0 (missing import `_`).
As this commit had no effect (due to the `size` parameter, I have reverted it in 8.0). It will be ported in 9.0 soon.

However this will not really fix your issue. You will get a unique constrain error instead of the traceback (better). This is because we don't allow duplicating currencies, as there is very few info on a currency (the rates are not duplicated), I don't see there is much use to duplicate a currency if you will need to change the name and symbol anyway...
